sdpoker.js

```diff
@@ -51,6 +52,8 @@ const args = yargs
   .describe('multicast', 'Connection addresses must be multicast.')
   .describe('unicast', 'Connection addresses must be unicast.')
   .describe('shaping', 'Check adherence to traffic shaping specification.')
+  .describe('verbose', 'Print passed tests as well as failed.')
```

Comment:
Once we're happy with the `verbose` mechanics, will we use also in _checkRFC4566.js_ etc.?

Reply:
Makes sense.  Likely some follow on work if we can put this off a bit so can get the 2110-22 tests committed into main.

Reply:
Any suggestions on how the verbose should be coded to make it cleaner? I'm not a javascript expert so my verbose comments have some duplication and difficulty in maintaining. I'd like to get these aded also to checkRFC4566 etc soon so need to decide format. Could put in the current method and add an item to go back an refactor to improve the verbose mechanism of we don't have a clear target example for code format.